freshmaker/events.py

```diff
@@ -452,3 +452,11 @@ def __init__(self, msg_id, dist_git_branch, container_images,
         self.brew_target = brew_target
         self.requester = requester
         self.requester_metadata_json = requester_metadata_json
+
+
+class BotasChangeStateEvent(ErrataBaseEvent):
```

Comment:
Since this event is only produced when the advisory is shipped, let's rename this event so it's a little clearer.
```suggestion
class BotasErrataShippedEvent(ErrataBaseEvent):
```